graphql-dgs-reactive/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
@paulbakker could we make it build.gradle.kts to keep it consistent?

Reply:
Huh I hadn't even noticed. Should be consistent for sure.